test/run/ch-run_squash.bats

```diff
@@ -0,0 +1,18 @@
+load ../common
+
+@test 'ch-run: run squashfs image' {
```

Comment:
I think this test is covered by the `make-auto` business above and can be removed.

Reply:
Agreed. I didn't orignally think I would add so much support for squashfs initially, so now that can be removed.